plrust/src/lib.rs

```diff
@@ -53,6 +53,22 @@ use pgx::{pg_getarg, prelude::*};
 pub use tests::pg_test;
 pgx::pg_module_magic!();
 
+/// This is the default set of lints we apply to PL/Rust user functions, and require of PL/Rust user
+/// functions before we'll load and execute them.
+///
+/// The defaults **can** be changed with the `plrust.compile_lints` and `plrust.required_lints` GUCS
+///
+/// This is
```

Comment:
```suggestion
/// This is an incomplete thought.
```

Reply:
I call this the "cliff hanger"... stay tuned for PL/Rust 2.0 for